File: app/src/main/java/ru/coffeeplanter/androidlogin/presentation/fragments/loggedin/LoggedInPresenter.java
package ru.coffeeplanter.androidlogin.presentation.fragments.loggedin;

import ru.coffeeplanter.androidlogin.presentation.BasePresenter;

interface LoggedInPresenter extends BasePresenter {

    void onFragmentCreateView();
    void exitButtonClicked();
    void forgetUserButtonClicked();
    void onFragmentDestroy();

}

File: app/src/main/java/ru/coffeeplanter/androidlogin/presentation/fragments/loggedin/LoggedInPresenter.kt
package ru.coffeeplanter.androidlogin.presentation.fragments.loggedin

import ru.coffeeplanter.androidlogin.presentation.BasePresenter

internal interface LoggedInPresenter : BasePresenter {

    fun onFragmentCreateView()
    fun exitButtonClicked()
    fun forgetUserButtonClicked()
    fun onFragmentDestroy()

}
